sbgECom/src/IO/Logger.cpp

```diff
@@ -13,7 +13,22 @@
 #include <chrono>
 #include <mutex>
 
+
+Logger::~Logger() {
+	fileStream->close();
+}
+
 void Logger::initialize() {
+	IO::initialize();
+}
+
+bool Logger::isInitialized() {
+    if (status.fileStatus == true)
+		return true;
+	return false;
```

Comment:
Should this be the following?

```suggestion
    return status.fileStatus == READY;
```

Reply:
Yes I corrected that earlier today